app/assets/javascripts/templates/stories/show.jst.ejs

```diff
@@ -166,7 +166,9 @@
       <div class="overall-task-progress">
         <h4>Task progress</h4>
         <div class="progress-bar">
-          <span class="completion"></span>
+          <span class="completion" style="width: {{ taskCompletionPercentage() }}%;">
```

Comment:
Yeah, not entirely sure how it works read it somewhere though that computed properties that are a function get evaluated each time the scope updates, pretty sweet.